assets/extensions/login-template.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Login</title>
+  </head>
+  <body>
+
+    {{ if .Error }}
+      <div>{{ .Error }}</div>
+    {{ end }}
+
+    <form action="{{ .Action }}" method="POST">
+      <input type="hidden" name="then" value="{{ .Values.Then }}">
+      <input type="hidden" name="csrf" value="{{ .Values.CSRF }}">
+
+      <label for="inputUsername">Username</label>
+      <input type="text" id="inputUsername" placeholder="" tabindex="1" autofocus="autofocus" type="text" name="username" value="{{ .Values.Username }}">
+
+      <label for="inputPassword">Password</label>
+      <input type="password" id="inputPassword" placeholder="" tabindex="2" type="password" name="password" value="">
+
+      <button type="submit" tabindex="4">Log In</button>
+
+    </form>
+
+  </body>
+</html>
```

Comment:
see create_bootstrap_project_template.go for a command example, though yours will be much simpler, since it doesn't have multiple output formats or any input parameters